runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -148,8 +149,29 @@
 })
 public class StreamingDataflowWorker {
 
-  private static final Logger LOG = LoggerFactory.getLogger(StreamingDataflowWorker.class);
+  // TODO(https://github.com/apache/beam/issues/19632): Update throttling counters to use generic
+  // throttling-msecs metric.
+  public static final MetricName BIGQUERY_STREAMING_INSERT_THROTTLE_TIME =
+      MetricName.named(
+          "org.apache.beam.sdk.io.gcp.bigquery.BigQueryServicesImpl$DatasetServiceImpl",
+          "throttling-msecs");
+  // Maximum number of threads for processing.  Currently each thread processes one key at a time.
+  static final int MAX_PROCESSING_THREADS = 300;
+  static final long THREAD_EXPIRATION_TIME_SEC = 60;
+  static final long TARGET_COMMIT_BUNDLE_BYTES = 32 << 20;
+  static final int MAX_COMMIT_QUEUE_BYTES = 500 << 20; // 500MB
+  static final int NUM_COMMIT_STREAMS = 1;
+  static final int GET_WORK_STREAM_TIMEOUT_MINUTES = 3;
+  static final Duration COMMIT_STREAM_TIMEOUT = Duration.standardMinutes(1);
```

Comment:
should the other constants be moved to the top here?
(if not, what is the motivation for moving some of them?)

Reply:
i think either the google java linter style or the spotless apply linter organizes constants in public, protected package private, then private order.

this was applied by spotless apply or google java formatter